PlaceholderSession/Views/TableViewCell/UserTableViewCell.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  UserTableViewCell.swift
+//  PlaceholderSession
+//
+//  Created by Maxos on 6/7/23.
+//
+
+import UIKit
+
+final class UserTableViewCell: UITableViewCell {
+    @IBOutlet private weak var nameLabel: UILabel!
+    @IBOutlet private weak var usernameLabel: UILabel!
+    @IBOutlet private weak var emailLabel: UILabel!
+    @IBOutlet private weak var phoneLabel: UILabel!
+    @IBOutlet private weak var websiteLabel: UILabel!
+    @IBOutlet private weak var companyLabel: UILabel!
+    @IBOutlet private weak var addressLabel: UILabel!
+    
+    var nameLabelText: String? {
```

Comment:
Better to create setup functions. All this didSets like copy of your hidden private vars. It is better to create function like setup(model: User), and setup all properties in this method. All these variables with didSets can be removed